Xlib/ext/dpms.py

```diff
@@ -141,13 +141,16 @@ class DPMSSetTimeouts(rq.Request):
 
 
 def set_timeouts(self, standby_timeout, suspend_timeout, off_timeout):
-    return DPMSSetTimeouts(display=self.display,
-                           opcode=self.display.get_extension_major(extname),
-                           major_version=1,
-                           minor_version=1,
-                           standby_timeout=standby_timeout,
-                           suspend_timeout=suspend_timeout,
-                           off_timeout=off_timeout)
+    DPMSSetTimeouts(display=self.display,
```

Comment:
Maybe it would be a better idea to return the result from this call (that is `None`) so nobody depends on this workaround.

WDYT :thinking: ?